### PR TITLE
build: Improve documentation

### PR DESCRIPTION
**Pull request by jhrozek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,12 @@ The Unit tests require the [cmocka](https://cmocka.org/) unit test
 framework as well as `nss_wrapper` and `pam_wrapper` tools from the
 [cwrap.org](https://cwrap.org/) project.
 
+If you're building from a git checkout, generate configure first:
+    $ autoreconf -if
+Then run the usual "configure, make, make install" commands. For platforms
+that we tested and support, there is a platform-dependant README.$platform
```
Comment (preichl) [inline README.md]: typo 's/dependant/dependent'

**Comment by preichl** [comment]
Just a small typo otherwise LGTM.

Should we also extend the list with hints of packages to install? I think I struggled to find out that gettext was required...

**Comment by jhrozek** [comment]
OK, I pushed a new version, does it look better now?

**Comment by preichl** [comment]
Yup, thanks. Fine by me.

**Comment by jhrozek** [comment]
Thanks for the review.

**Pull request merged**